test/monkeypatch.mjs

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env -S node --experimental-modules
+
+import Demitasse from '@pipobscure/demitasse';
+const { describe, it, report } = Demitasse;
+
+import Pretty from '@pipobscure/demitasse-pretty';
+const { reporter } = Pretty;
+
+import { strict as assert } from 'assert';
+const { equal } = assert;
+
+import * as Temporal from '@js-temporal/polyfill';
+const { PlainDate } = Temporal;
+
+describe('Monkeypatch', () => {
+  describe('PlainDate', () => {
+    it("Monkeypatching Duration doesn't affect PlainDate", () => {
+      Temporal.Duration.prototype.constructor = null;
```

Comment:
Probably we also want to check `Temporal.Duration = null` and `delete Temporal.Duration`

Reply:
As the tests are `.mjs`, imports will be read-only, so JS won't let us do those two things (although monkeypatching the prototype seems to be valid).